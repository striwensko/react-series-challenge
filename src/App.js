import React, { Component } from 'react';
class App extends Component {
  render() {
    return (
      
        <div>
          
            <h1>Giphy Challenge</h1>
            <ul>
              <li>home</li>
              <li>favorites</li>
            </ul>
          

        </div>
      
    );
  }
}

export default App;
