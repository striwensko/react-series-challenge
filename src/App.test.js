import React from 'react';
import { shallow } from 'enzyme';
import App from './App';


describe('Favorites tests',()=>{
    it('render',()=>{
        const component = shallow(<div></div>);
        
        expect(component).toEqual(1);
       
        
    })
})
