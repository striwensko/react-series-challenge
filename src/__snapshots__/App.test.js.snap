// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites tests render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Giphy Challenge
</h1>,
        <ul>
          <li>
                    home
          </li>
          <li>
                    favorites
          </li>
</ul>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Giphy Challenge",
        },
        "ref": null,
        "rendered": "Giphy Challenge",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              home
</li>,
            <li>
              favorites
</li>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "home",
            },
            "ref": null,
            "rendered": "home",
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "favorites",
            },
            "ref": null,
            "rendered": "favorites",
            "type": "li",
          },
        ],
        "type": "ul",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Giphy Challenge
</h1>,
          <ul>
            <li>
                        home
            </li>
            <li>
                        favorites
            </li>
</ul>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Giphy Challenge",
          },
          "ref": null,
          "rendered": "Giphy Challenge",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                home
</li>,
              <li>
                favorites
</li>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "home",
              },
              "ref": null,
              "rendered": "home",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "favorites",
              },
              "ref": null,
              "rendered": "favorites",
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
